### DISCUSSION TITLE
forget my account and password

### DISCUSSION THREAD

**Discussion by spurbro** [LizardByte/community#186] [Sunshine] [answered]
i forget my account and password of my sunshine, now i want to connect new equipment, how can i do?

**Comment by ld892012** [comment, marked as answer]
Maybe try [This](https://docs.lizardbyte.dev/projects/sunshine/en/latest/troubleshooting/general.html), you can create new user/password and access again to webui

**Reply by weaselmuncher** [reply]
Can someone help me, ive tried the below suggestions and nothing is working 

At line:1 char:44
+ "C:\Program Files\Sunshine\sunshine.exe" --creds weaselmuncher weasel ...
+                                            ~~~~~
Unexpected token 'creds' in expression or statement.
At line:1 char:1
+ "C:\Program Files\Sunshine\sunshine.exe" --creds weaselmuncher weasel ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The '--' operator works only on variables or on properties.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken

**Reply by ReenigneArcher** [reply]
@weaselmuncher use command prompt

**Reply by weaselmuncher** [reply]
i am

**Reply by weaselmuncher** [reply]
no I wasn't I lied, had to run as admin and think its worked. thanks for the help guys :)

**Reply by ReenigneArcher** [reply]
Yea, that output is from PowerShell, not Command Prompt

**Reply by weaselmuncher** [reply]
C:\Windows\System32>"C:\Program Files\Sunshine\sunshine.exe" --creds weaselmuncher weaselmuncher
[2023:12:09:22:10:23]: Info: New credentials have been created

Ive got this msg however, the localhost isnt accepting my new creds, perhaps i need to wait

**Reply by weaselmuncher** [reply]
its worked, im on. I really need to get better with all this computer stuff aha. thanks again for your help :)

**Reply by ghost** [reply]
where shuld i run this?

**Comment by NanoTheGlitch** [comment]
Where would I be writing this? I cant type in the sunshine window itself and cmd isnt recognising the command

**Reply by ld892012** [reply]
it's a command to type with cmd and try to add the Sunshine's path to the prompt ("C:\Program Files\Sunshine" i think) to give something like this :
`"C:\Program Files\Sunshine\sunshine.exe" --creds <new username> <new password>`

**Reply by NanoTheGlitch** [reply]
> it's a command to type with cmd and try to add the Sunshine's path to the prompt ("C:\Program Files\Sunshine" i think) to give something like this : `"C:\Program Files\Sunshine\sunshine.exe" --creds <new username> <new password>`

That Worked! thank u so much!

**Reply by spurbro** [reply]
when i try to do it, it shows shouldnt have<, there something wrong?

**Reply by ld892012** [reply]
Can you give some more information about your problem ?

**Reply by ReenigneArcher** [reply]
```bat
cd %programfiles%/Sunshine
sunshine.exe --creds your_new_username your_new_password
```

`<>` are common to use to indicate placeholders... you don't actually include the brackets

**Reply by spurbro** [reply]
think you, i have done it！

**Reply by SolarGlider** [reply]
I'm a complete noob with these things. I installed the latest version of sunshine and saved my log-in credentials in a password manager. However when I try to access the UI through the web I can't access it anymore. I've tried using the commands suggested above but to no avail. When sunshine is running you can open a CMD tab to enter commands but I receive this.

![image](https://user-images.githubusercontent.com/132139770/235300572-d36c9377-ad0d-4859-9b5f-50edc714ec8e.png)

I have no idea what I'm doing wrong here. I've tried resetting through the installation folders of sunshine but that doesn't work either. 
Could anyone help please since I'm very stuck. Thanks in advance!

**Reply by SolarGlider** [reply]
I just managed to login with username sunshine and my password, but I assume this is not going to help me either? Resetting the password through the UI claims I entered the wrong credentials for a reset.

**Reply by SolarGlider** [reply]
After some messing around it seems I got it working again. I just entered the pin in moonlight. Now I just have the issue of port forwarding.

**Reply by kingchm** [reply]
I have tried the command for new username and password but every time I try iT nothing happens.

**Reply by ReenigneArcher** [reply]
`cd /d %programfiles%/Sunshine`

`sunshine --creds your_new_username your_new_password`

**Reply by kingchm** [reply]
Do I type it in just like that?

**Reply by kingchm** [reply]
Nvm it just took time to resolve it

**Comment by knifeknerd** [comment]
How is it possible for this thing to forget me 2 hours after I just set it up? username and password that I defined does not work. No password reset, no login, I'm not a coder. lol... How about dumbing it down for the low level nerds.

**Reply by ReenigneArcher** [reply]
You don't need to be a coder to do this. Open your terminal, cd to the path where sunshine is, enter the command.

**Reply by knifeknerd** [reply]
> You don't need to be a coder to do this. Open your terminal, cd to the path where sunshine is, enter the command.

I must not speak whatever language it is you guys are writing this command in. Anyways I just deleted the install and created all new credentials, we will see if it takes this time. It just doesnt make any sense to me that I set it up, created credentials, went and played with it for an hour, then after restarting my PC it's no longer logged in to the account I created and no longer accepts the credentials I just created.
